git-remote-hg

```diff
@@ -332,7 +332,7 @@ class BranchMap(object):
                 sha1 = store.changeset_ref(head)
                 if not sha1:
                     continue
-                extra = store.read_changeset_data(sha1).get('extra')
+                extra = store.read_changeset_data(sha1)
```

Comment:
This breaks the code that follows. But read_changeset_data shouldn't be returning None in the first place. What is the output of git cinnabar fsck?